src/app/shared/store/provider.actions.ts

```
@@ -472,3 +473,69 @@ export class OnGetPositionByIdFail {
   static readonly type = '[provider] Get position by id fail';
   constructor(public error: HttpErrorResponse) {}
 }
+
+export class GetLanguageList {
+  static readonly type = '[provider] get language list';
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)